com.unity.cinemachine/Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -1012,14 +1012,14 @@ void PushStateToUnityCamera(ref CameraState state)
                 Camera cam = OutputCamera;
                 if (cam != null)
                 {
-                    cam.ResetProjectionMatrix();
+                    bool isPhysical = cam.usePhysicalProperties;
+                    if (!isPhysical)
+                        cam.ResetProjectionMatrix();
```

Comment:
ResetProjectionMatrix is there for a reason.  You can't just not call it just because camera is physical.

Reply:
But it sets the camera to not physical. Do you know why? We are not setting the cam.projectionMatrix anywhere, so I don't understand why it sets the camera back to not physical.

Now I am checking if physical, then restore it after reseting the projection matrix.

Reply:
it's because `ResetProjectionMatrix` behaves badly: it resets `usePhysicalProperties` when it shouldn't..  I checked with the camera team.

Reply:
Thanks for checking! :)